src/relay/backend/task_extraction.cc

```diff
@@ -31,25 +31,57 @@ namespace tvm {
 namespace relay {
 namespace backend {
 
-namespace metaschedule {
-
-using meta_schedule::ExtractedTask;
+bool DefaultTaskFilter(const Array<te::Tensor>& args) {
+  using namespace ::tvm::te;
+  std::vector<Tensor> stack;
+  std::unordered_set<const TensorNode*> visited;
+  for (const Tensor& v : args) {
+    for (const PrimExpr& e : v->shape) {
+      if (!e->IsInstance<IntImmNode>()) {
+        return false;
```

Comment:
Better to have some comments in this function, such as "doesn't support dynamic shape", etc.

Reply:
done!